notebooks/2021-09/2021-09-07/sat_data.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+
+from nowcasting_dataset.data_sources.satellite_data_source import SatelliteDataSource
+
+s = SatelliteDataSource(
+    filename="gs://solar-pv-nowcasting-data/satellite/EUMETSAT/SEVIRI_RSS/OSGB36/"
+    "all_zarr_int16_single_timestep.zarr",
+    history_len=6,
```

Comment:
Yea, I was debating if to go back and fix notebooks. It's nice if they are all working, but it also means a lot more code to maintain if they are not. Part of me thinks if it needs maintaining it should be move away from a notebook into the package, and that will need updating. What do you think? I also don't really know what the "common" practice is here

Reply:
I would go with, since this is more of a library to build things off of, that if it should be maintained, it should be in the package and not a notebook, and otherwise if its in a notebook, it'll be updated from time to time, but not necessarily maintained much.

Reply:
Yeah, I agree - let's deliberately let notebook code 'rot' a bit.  I'll add a note to the coding style doc!

IIRC, there are ways to automatically test notebooks (by testing if it always produces the same outputs) but let's not worry about this for now.

If our code starts to become used by, say, 1,000 datathon participants then we might have to start paying more attention to our notebooks :) ... but let's cross that bridge if it's built :)